src/instructlab/configuration.py

```diff
@@ -1336,3 +1486,32 @@ def storage_dirs_exist() -> bool:
         DEFAULTS.PHASED_DIR,
     ]
     return all(os.path.exists(dirpath) for dirpath in dirs_to_check)
+
+
+# render_configs_and_profiles takes the amount of GPUs on the system and returns a list of the following format:
+# [(MAX_VRAM_FOR_THIS_GROUP, [{"TRAIN_PROFILE_NAME": (VRAM_MAX_FOR_PROFILE, TRAIN_PROFILE)}....])]
+def render_profiles() -> list[tuple[int, list[dict[str, tuple[int, _train]]]]]:
+    return [
+        (
+            16,
+            [{"NVIDIA RTX 3070/3080/4070/4080": (16, SINGLE_CONSUMER_GPU_TRAIN)}],
```

Comment:
What about the 4090 model, it has 24 GB and presumably would be something a high end desktop would have.